inverted_pendulum:

  # Publish joint states
  joint_state_publisher:
    type: joint_state_controller/JointStateController
    publish_rate: 100
  
  joint_cart_controller:
    type: effort_controllers/JointEffortController
    joint: joint_ref_cart
    publish_rate: 200

  gazebo_ros_control:
     pid_gains:

       joint_ref_cart:
         p: 30
         i: 24
         d: 2



  